MEDO v0.5.owl

```diff
@@ -804,6 +975,536 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
+        <rdfs:label>informal summative oral patient presentation</rdfs:label>
+        <skos1:related>MedBiquitous - AM11: Oral Patient Presentation</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#informal_summative_participation_assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#informal_summative_participation_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000107"/>
+        <obo:IAO_0000115>A type of informal summative assessment that evaluates the student&apos;s attendance, participation, engagement, and performance in an activity.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM12: Sharing or taking part in an activity (ERIC, 1966b) 
+
+Synonymous with/Includes: Attendance</obo:IAO_0000119>
+        <rdfs:label>informal summative participation assessement</rdfs:label>
+        <skos1:related>MedBiquitous - AM12: Participation</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#informal_summative_peer_assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#informal_summative_peer_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000107"/>
+        <obo:IAO_0000115>A type of informal summative assessment where fellow learner(s) observes, assesses, and provides written or oral subjective feedback of a learner&apos;s work or performance.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM13: The concurrent or retrospective review by learners of the quality and efficiency of practices or services ordered or performed by fellow learners.
+
+Does not include: Multisource Assessment; Narrative Assessment</obo:IAO_0000119>
+        <rdfs:label>informal summative peer assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM13: Peer Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#informal_summative_self-assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#informal_summative_self-assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000107"/>
+        <obo:IAO_0000115>A type of informal summative assessment where student observes, evaluates, and reflects upon one’s own deficiencies, achievements, behavior or professional performance, and competencies. The assessment is completed by the learner to reflect upon and critically assess their own performance against a set of established criteria.</obo:IAO_0000115>
```

Comment:
My revision: "A type of informal summative assessment in which a student observes, evaluates, and reflects upon the student's own deficiencies, achievements, behavior or professional performance, and competencies. The assessment is completed by the student to reflect upon and critically assess the student's own performance against a set of established criteria."